features/poll/api/src/main/kotlin/io/element/android/features/poll/api/PollContentView.kt

```diff
@@ -157,6 +195,31 @@ private fun ColumnScope.UndisclosedPollBottomNotice(
     )
 }
 
+@Composable
+private fun CreatorView(
+    @Suppress("SameParameterValue") votesCount: Int, // TODO Polls: remove @Suppress when edit poll screen is implemented.
+    isPollEnded: Boolean,
+    onPollEdit: () -> Unit,
+    onPollEnd: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    if (!isPollEnded) {
+        if (votesCount == 0) {
+            Button(
+                text = "Edit Poll",
```

Comment:
Localazy? Also maybe `Edit poll` is better, not sure.

Reply:
Thanks for noticing, it's a leftover.